utils/gen_onnx_mlir.py

```diff
@@ -1441,7 +1432,6 @@ def main(args):  # type: (Type[Args]) -> None
                     op_def.write(r)
                     previous_name = schema.name
 
-    gen_op_new_version(op_importer, new_version_dict)
```

Comment:
removed this because I couldn't see any other uses after I removed its use in GetImportVersionOfNode in FrontendDialectTransformer.cpp